components/Header/Header.js

```diff
@@ -10,3 +10,17 @@ export default function Header() {
     </>
```

Comment:
You don't need a fragment `<>`element at this point. The `<StyledHeader>`already nest the other tags.

Reply:
agree & Thanks for your time double checking 🙌